src/pages/publication/ReferencesPanel.tsx

```diff
@@ -31,42 +31,44 @@ interface ReferencesPanelProps {
 const ReferencesPanel: React.FC<ReferencesPanelProps> = ({ goToNextTab, savePublication }) => {
   const { t } = useTranslation('publication');
   const { values, setFieldTouched, setFieldValue }: FormikProps<FormikPublication> = useFormikContext();
-  const { publicationType } = values.entityDescription;
-
-  // Validation messages won't show on fields that are not touched
-  const setAllFieldsTouched = useCallback(() => {
-    Object.values(ReferenceFieldNames).forEach((fieldName) => setFieldTouched(fieldName));
-  }, [setFieldTouched]);
+  const { publicationType, publicationSubtype } = values.entityDescription;
 
   useEffect(
-    () => () => {
-      // Set all fields as touched if user navigates away from this panel (on unmount)
-      setAllFieldsTouched();
-    },
-    [setAllFieldsTouched]
+    // Set all fields as touched if user navigates away from this panel (on unmount)
+    () => () => Object.values(ReferenceFieldNames).forEach((fieldName) => setFieldTouched(fieldName)),
+    [setFieldTouched]
   );
 
+  useEffect(() => {
+    // Update publicationInstance's type when publicationSubtype changes
```

Comment:
Snakket med Rurik om å fjerne publicationSubtype fra modellen siden den uansett skal være lik instance type, så det kommer antageligvis til å skje (en gang)

Reply:
er publicationSubType `article`, `short communication` osv? hva mener du med at det uansett er lik instance type?

Reply:
publicationSubtype = publicationInstance.type = "JournalArticle" | "ShortCommunication" | etc

Reply:
Det minner meg på at det er noen nye verdier for publicationType også... Gamle "JournalArticle" er nå "JournalContent"

Reply:
kan `publicationSubType` fjernes fra vår modell uten at det fucker opp lagringen? Isåfall så tenker jeg vi bare kan gjøre det nå kanskje? Hvis publicationInstance.type er på plass da?

Reply:
`publicationSubType` kreves fortsatt av backend ja